plugins/seedwatch.cpp

```diff
@@ -180,8 +181,13 @@ DFhackCExport command_result plugin_load_data (color_ostream &out) {
     World::GetPersistentData(&seed_configs, SEED_CONFIG_KEY_PREFIX, true);
     const size_t num_seed_configs = seed_configs.size();
     for (size_t idx = 0; idx < num_seed_configs; ++idx) {
-        auto &c = seed_configs[idx];
-        watched_seeds.emplace(get_config_val(c, SEED_CONFIG_ID), c);
+        auto& c = seed_configs[idx];
+        int seed_id = get_config_val(c, SEED_CONFIG_ID);
+        auto plant = binsearch_in_vector(world->raws.plants.all, &df::plant_raw::index, seed_id);
```

Comment:
could you separate this out into a function and tag it with a comment saying that we can remove the function once saves from 50.09 are no longer loadable?

Reply:
done